config/manager/default-config/db-statefulset.yaml

```diff
@@ -27,8 +27,9 @@ spec:
             - name: PGDATA
               value: /var/lib/pgsql/data/userdata
           envFrom:
-            - secretRef:
-                name: <POSTGRESQL_SECRET>  # will be replaced with 'backstage-psql-secrets-<cr-name>'
+#            - secretRef:
```

Comment:
Why this is removed?

Reply:
this declaration is not used. Do you see any value keeping it ?

Reply:
I guess it is not, it works well for me. Could you elaborate?